src/class/video/video_device.c

```diff
@@ -1283,6 +1290,24 @@ uint16_t videod_open(uint8_t rhport, tusb_desc_interface_t const * itf_desc, uin
     cur = _next_desc_itf(cur, end);
     stm->desc.end = (uint16_t) ((uintptr_t)cur - (uintptr_t)itf_desc);
     stm->state = VS_STATE_PROBING;
+#ifdef TUP_DCD_EDPT_ISO_ALLOC
+    /* Allocate ISO endpoints */
```

Comment:
Is it possible that one VS interface has multiple endpoints, one input header and one output header ?

Reply:
Based on my understanding, one VS interface can have up to 2 endpoints, but one VS can only have one input header or one output header. Cite the standard.
![image](https://github.com/hathach/tinyusb/assets/45088311/bbae157c-e0d5-4aff-9e61-20ac6effa595)

When a VS interface has two endpoints, one is used for streaming video, and the other is used for capturing still images. Cite the standard.
![image](https://github.com/hathach/tinyusb/assets/45088311/07a124e2-5e7c-4577-9e53-2981401832f4)
 The case is method 3 shown below.
![image](https://github.com/hathach/tinyusb/assets/45088311/9bbe52e4-dda3-4c6b-a78a-ac25ff162020)

Reply:
Great, so it's safe to search for only 1 iso endpoint.